CMakeLists.txt

```diff
@@ -59,7 +59,7 @@ set(IGN_MSGS_VER ${ignition-msgs7_VERSION_MAJOR})
 #--------------------------------------
 # Find ignition-common
 # Always use the profiler component to get the headers, regardless of status.
-ign_find_package(ignition-common3 VERSION 3.5
+ign_find_package(ignition-common3 VERSION 3.6
```

Comment:
Is this version bump also for fixing warnings?

Reply:
It's actually failing the build fast... I used a new machine and noticed that building against common 3.5 fails because `ign-gazebo` is using the API from https://github.com/ignitionrobotics/ign-common/pull/70